app/views/providers/address_selections/show.html.erb

```diff
@@ -1,15 +1,15 @@
-<%= form_with(
-      model: @form,
-      scope: :address_selection,
-      url: providers_legal_aid_application_address_selection_path(@legal_aid_application),
-      method: :patch,
-      local: true
-    ) do |f| %>
-
-  <%= page_template(
-        page_title: t('forms.address_selection.heading'),
-        template: :form
-      ) do %>
+<%= page_template(
+      page_title: t('forms.address_selection.heading'),
+      template: :form
+    ) do %>
+
+  <%= form_with(
+        model: @form,
+        scope: :address_selection,
+        url: providers_legal_aid_application_address_selection_path(@legal_aid_application),
+        method: :patch,
+        local: true
+      ) do |f| %>
```

Comment:
This reverses a change made in https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/359

Reply:
Does this change keep the structure of the HTML that #359 fixed?  
i.e.
`<form> <fieldset> <legend> <h1>`

Reply:
No it reverses that .... but the #359 fix only corrects that on one page. It leaves every other instance of the form page template behaving the same way. 

This issue needs to be raised in a separate ticket and fixing everywhere it occurs -which probably means a rethink of the way the form page template is used.

Reply:
Yes, okay.  That sounds sensible.